### ISSUE TITLE
Screen Readers don't announce how to get to Balloon Toolbars for widgets

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor5#11959]
## 📝 Provide detailed reproduction steps (if any)

1. Enable screen reader like JAWS
2. Enter the table

### ✔️ Expected result

Screen Reader announces how to get to the widget toolbar

### ❌ Actual result

Screen reader doesn't announce how to get to the widget toolbar. 

### ❓ Possible solution

_If you have ideas, you can list them here. Otherwise, you can delete this section._

## 📃 Other details

This issue is related to https://github.com/ckeditor/ckeditor5/issues/5146

* Browser: Any
* OS: Any
* First affected CKEditor version: N/A
* Installed CKEditor plugins: widget, table, image

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
